dev_guide/pod_autoscaling.adoc

```diff
@@ -68,7 +68,7 @@ For replication controllers, this scaling corresponds directly to the replicas
 of the replication controller. For deployment configurations, this will scale
 the latest deployment, or, if no deployment is present, the replication
 controller template in the deployment configuration. Note that scaling will
-occur if the latest deployment is marked as `Complete`.
+only occur if the latest deployment is marked as `Complete`.
```

Comment:
When the autoscaler targets a deploymentConfig, it should only be setting `deploymentConfig.spec.replicas`.

>For deployment configurations, this will scale the last Complete deployment, 

This is true, but only indirectly: the scaler sets `spec.replicas` on the deploymentConfig, causing the latest deployment to be scaled by another mechanism (the deployment controllers). Not sure if that matters in the context of this documentation, but felt it worth clarifying on this thread.

>or, if no Complete deployment is present, the replication controller template in the deployment configuration.

The deploymentConfig has a spec (which is a deploymentConfigSpec with a replicas field) and a podTemplateSpec (which isn't a replication controller template).

The way I would have worded this is: "For deployment configurations, scaling corresponds directly to the replica count of the deployment configuration." and go from there if you need to explain how changing that field affects deployment scaling.